src/Website/Directory.php

```diff
@@ -102,6 +102,16 @@ public function path(string $path = null, $local = false): string
         return $path;
     }
 
+    public function putFile($path, $file = null, $options = [])
+    {
+        // tba
+    }
+
+    public function putFileAs($path, $file, $name = null, $options = [])
+    {
+        // tba
+    }
```

Comment:
@mikebronner Maybe the only thing that still needs to be added are these methods to make sure nobody's application actually fails. Other than that: I'm fine with merging this